### ISSUE TITLE
add UI to report bad sites and block "unblocked" ads

### ISSUE THREAD

**Issue by bradleyrichter** [brave/browser-laptop#1370]
from zendesk request:

My bug is ... Need way to block ads found, report scam sites.

**Comment by alexwykoff** [comment]
cc @bsclifton 

assigning myself to make a prototype pronto

**Comment by bsclifton** [comment]
Closing in favor of https://github.com/brave/brave-browser/issues/777
